.gitignore

```diff
@@ -0,0 +1 @@
+lychee
```

Comment:
This is the .gitignore, so won't effect what's added to the Docker image. Adding to the .dockerignore might be more useful, but I have no objection to either.

Reply:
yeah but I was in the mindset of people using the docker compose instead of docker build. :)

Reply:
Ok. I have almost no experience there :)